nav_msgs/Path query_path
---
nav_msgs/Path safe_path
